src/i2c.rs

```diff
@@ -253,12 +253,12 @@ macro_rules! hal {
                     pins: PINS,
                     mode: Mode,
                     clocks: Clocks,
-                    apb: &mut APB1,
```

Comment:
Is this a breaking change?

If I understand it correctly, it shouldn't be even though the type of a public function is changing

Reply:
No it is not breaking change (it is still `APB1`).